scheduler/plugin/guest.go

```diff
@@ -98,3 +94,26 @@ func decorateError(out fmt.Stringer, fn string, err error) error {
 	}
 	return err
 }
+
+type exports uint
+
+const (
+	exportFilterPlugin exports = 1 << iota
+)
+
+func detectExports(exportedFns map[string]wazeroapi.FunctionDefinition) (exports, error) {
+	var e exports
+	for name, f := range exportedFns {
+		switch name {
+		case guestExportFilter:
```

Comment:
as we add new plugins, we add detection logic here. Some plugins might be composite because the developer wants to implement both filter and score for better performance vs defining multiple plugins.